pkg/cmd/pipeline/describe.go

```diff
@@ -76,9 +76,9 @@ const describeTemplate = `{{decorate "bold" "Name"}}:	{{ .PipelineName }}
 {{- $tl := len .Pipeline.Spec.Tasks }}{{ if eq $tl 0 }}
  No tasks
 {{- else }}
- NAME	TASKREF	RUNAFTER
+ NAME	TASKREF	TIMEOUT	RUNAFTER	PARAMS
```

Comment:
Maybe `TIMEOUT` should be the column after `RUNAFTER` as I see `RUNAFTER` as a more significant piece of information. I do think it makes sense to have `PARAMS` for the last column to provide some space for it. WDYT?

Reply:
Yes, that would be fine `TIMEOUT` after `RUNAFTER` and `PARAMS` at last as it would take more space depending on the number of params.